src/modules/claims/claims.service.ts

```diff
@@ -1227,8 +1235,11 @@ export class ClaimsService {
    *
    * @param {String} role Registration types of the claim
    */
-  private async verifyIssuer(role: string): Promise<void> {
-    await this._issuerVerification.verifyIssuer(this._signerService.did, role);
+  private async verifyIssuer(role: string): Promise<VerificationResult> {
```

Comment:
There is not much sense in this method now, because it is just forwards call. How about throwing here instead of in calling code?